api/v1beta1/glance_webhook.go

```diff
@@ -33,9 +33,15 @@ var glanceDefaults GlanceDefaults
 // log is for logging in this package.
 var glancelog = logf.Log.WithName("glance-resource")
 
-// SetupDefaults - initialize Glance spec defaults for use with either internal or external webhooks
+// SetupDefaults - FIXME: remove in followup PR to satisfy CI for this PR
```

Comment:
I'm assuming this is needed to make the CI happy (which is what you described in the other comment).
In that case I'm fine about having a follow up where this is going to be removed.

Reply:
Yes.  What was happening is that the CI was updating OpenStack operator to use this version of Glance, which then broke the OpenStack operator because it still has old code for using the old defaults interface [1].  There are a few operators that will need this approach to be updated to the new interface without breaking CI.  Then once all of those service operators are changed, we can go into OpenStack operator and finalize the calls to the default interfaces.

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/d370c7f9a716b2f723799b6a35d4200fc63d3393/main.go#L192